docs/snippets/common/storybook-upgrade-prerelease.pnpm.js.mdx

```diff
@@ -1,3 +1,3 @@
 ```shell
-pnpm dlx storybook@latest upgrade --prerelease
+pnpx storybook@latest upgrade --prerelease
```

Comment:
`pnpx` has been deprecated in favor of `pnpm dlx`. You can't really see it in the docs anymore, because `pnpx` hasn't been mentioned since v6, whose docs are no longer public. But you can check the wayback machine here: http://web.archive.org/web/20230202163724/https://pnpm.io/6.x/pnpx-cli
```suggestion
pnpm dlx storybook@latest upgrade --prerelease
```